CHANGES.txt

```diff
@@ -11,4 +11,4 @@ v1.2.5, 2022-07-12 -- Block some more bot useragents
 v1.2.6, 2022-07-13 -- Block one more useragent - "gh"
 v1.2.7, 2022-07-15 -- Block more user agents - "Petalbot"
 v1.3.0, 2023-02-20 -- Add rate limits
-v1.4.0, 2-24-06-21 -- Migrate to Flask.Limiter for rate limits
+v2.0.0, 2-24-06-21 -- Migrate to Flask.Limiter for rate limits. Breaking Change: 'app' must be passed to 'build_search_view' as the positon 1 argument
```

Comment:
1.4.0 was released, so maybe keep it?

Also, there is a typo in the date year. And I guess the date is not true anymore?